Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/AppDatabase.kt

```diff
@@ -67,7 +71,7 @@ abstract class AppDatabase : RoomDatabase() {
                  * The fallback behavior is to reset the app as we only store exposure summaries
                  * and cached references that are non-critical to app operation.
                  */
-                .fallbackToDestructiveMigrationFrom()
+                .fallbackToDestructiveMigration()
```

Comment:
Why the unrelated change?

Reply:
as you remember the bug data was first hold in a table inside the old database. As requested, I moved it to a separate DB, after removing my code from this class I had to mop up. This is clean autoformat.

Reply:
the second part of the file is really unrelated (fallback). 
Removed.